packages/apollo-server-client/index.js

```diff
@@ -3,15 +3,23 @@ const { BatchHttpLink } = require('@apollo/client/link/batch-http')
 const { RetryLink } = require('@apollo/client/link/retry')
 const { onError }  = require('apollo-link-error')
 const { createUploadLink } = require('apollo-upload-client')
-const fetchPoltfill = require('cross-fetch/polyfill')
+const fetchPolyFill = require('cross-fetch/polyfill')
```

Comment:
We already have node-fetch. Should we add it to package.json ?

Reply:
node-fetch is not working with Apollo client on server side (as it is not provided poly fill entry point)
so to use node-fetch we need to add
```
const fetch = require('node-fetch')

if (!globalThis.fetch) {
    globalThis.fetch = fetch
}
```
to have fetch available globally